docs_src/microservices/application/ContextAPI.md

```diff
@@ -1,67 +1,94 @@
 The context parameter passed to each function/transform provides operations and data associated with each execution of the pipeline. Let's take a look at a few of the properties that are available:
 
 ```go
-type Context struct {
-	// ID of the EdgeX Event (will be filled for a received JSON Event)
-	EventID string
-	
-	// Checksum of the EdgeX Event (will be filled for a received CBOR Event)
-	EventChecksum string
-	
-	// This is the ID used to track the EdgeX event through entire EdgeX framework.
-	CorrelationID string
-	
-	// OutputData is used for specifying the data that is to be outputted. Leverage the .Complete() function to set.
-	OutputData []byte
-	
-	// This holds the configuration for your service. This is the preferred way to access your custom application settings that have been set in the configuration.	
-	Configuration common.ConfigurationStruct
-	
-	// LoggingClient is exposed to allow logging following the preferred logging strategy within EdgeX.
-	LoggingClient logger.LoggingClient
-	
-	// EventClient exposes Core Data's EventClient API
-	EventClient coredata.EventClient
-	
-	// ValueDescriptorClient exposes Core Data's ValueDescriptor API
-	ValueDescriptorClient coredata.ValueDescriptorClient
-	
-	// CommandClient exposes Core Commands's Command API
-	CommandClient command.CommandClient
-	
-	// NotificationsClient exposes Support Notification's Notifications API
-	NotificationsClient notifications.NotificationsClient
-	
-	// RetryData holds the data to be stored for later retry when the pipeline function returns an error
-	RetryData []byte
-	
-	// SecretProvider exposes the support for getting and storing secrets
-	SecretProvider *security.SecretProvider
-
-	// ResponseContentType sets a custom response type
-	ResponseContentType string
+// AppFunctionContext defines the interface for an Edgex Application Service Context provided to
+// App Functions when executing in the Functions Pipeline.
+type AppFunctionContext interface {
+    // CorrelationID returns the correlation ID associated with the context.
```

Comment:
These comments can be removed since they are redundant with the descriptions below.